openfisca_survey_manager/scenarios.py

```diff
@@ -1433,6 +1435,7 @@ def _apply_modification(self, simulation, period):
 
         def set_variable(varying_variable, varying_variable_value, period_):
             delta = self.variation_factor * varying_variable_value
+            delta = np.where(varying_variable_value == 0.0 ,1 , np.sign(delta)) * max_(100.0, abs_(delta))
```

Comment:
We should introduce an attribute to the scenario class to control for the minimal value which is arbitrary set to 100 here.